dipy/workflows/io.py

```diff
@@ -205,3 +206,100 @@ def run(self, data_names, out_dir=''):
             # the same process, we don't have the env variable pointing to the
             # wrong place
             self.load_module('dipy.data.fetcher')
+
+
+class NiftisToPamFlow(Workflow):
+
+    @classmethod
+    def get_short_name(cls):
+        return 'niftis_to_pam'
+
+    def run(self, peaks_dir_files, peaks_values_files, peaks_indices_files,
+            shm_files, gfa_files, out_dir='', out_pam="peaks.pam5"):
+        """Convert pam5 files to mutiple nifti files.
+
+        Parameters
```

Comment:
are all niftis mandatory in the pam format?

Reply:
yes!